.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+*.csv
```

Comment:
only Result files should not be in this PR. The test files should be

Reply:
done